src/cmd/run.js

```diff
@@ -257,14 +262,21 @@ export default async function run(
       }
 
       log.info('The extension will reload if any source file changes');
+      const createWatcher = (...args) => {
```

Comment:
ah, yes of course! OK, let's try to land https://github.com/mozilla/web-ext/pull/751 first so you can merge it into this branch when it hits master.